### ISSUE TITLE
Review code and issues from sourceforge bugs

### ISSUE THREAD

**Issue by skyghis** [skyghis/j-interop-ng#2]
Review code and issues from [Sourceforge bugs](https://sourceforge.net/p/j-interop/bugs/) :

- [ ] [#2](https://sourceforge.net/p/j-interop/bugs/2/)   UTF-16LE encoding corrupts binary data
- [x] ~~[#9](https://sourceforge.net/p/j-interop/bugs/9/)   Missing message~~ - Wont fix
- [x] ~~[#10](https://sourceforge.net/p/j-interop/bugs/10/) JIInterfacePointer should be public ?~~ - Won't fix
- [ ] [#11](https://sourceforge.net/p/j-interop/bugs/11/) DCOM connection does not attempt to resolve hostnames
- [ ] [#12](https://sourceforge.net/p/j-interop/bugs/12/) Connection to MS Outlook causes a win32 exception
- [ ] [#13](https://sourceforge.net/p/j-interop/bugs/13/) Test_ITestServer2_Impl crashs
- [x] [#15](https://sourceforge.net/p/j-interop/bugs/15/) Recognize response codes 0xc0000008 and 0xc0000034
- [ ] [#16](https://sourceforge.net/p/j-interop/bugs/16/) thread jI_ShutdownHook NullPointerException
- [ ] [#18](https://sourceforge.net/p/j-interop/bugs/18/) Incorrect byte length for JIMarshalUnMarshalHelper$ShortImpl
- [ ] [#19](https://sourceforge.net/p/j-interop/bugs/19/) JIVariant missing byte constructor (VT_I1)
- [ ] [#20](https://sourceforge.net/p/j-interop/bugs/20/) win7 localhost JIComServer.init () bug
- [ ] [#21](https://sourceforge.net/p/j-interop/bugs/21/) Socket Closed - OPC Server
- [x] [#22](https://sourceforge.net/p/j-interop/bugs/22/) destroySession blocked by Release_Reference_TImerTask
- [ ] [#23](https://sourceforge.net/p/j-interop/bugs/23/) Possible errors
- [x] ~~[#24](https://sourceforge.net/p/j-interop/bugs/24/) Licensing bugs~~ - Wont fix, not using this branch
